.github/workflows/cc-payload-after-push.yaml

```diff
@@ -0,0 +1,82 @@
+name: CI | Publish Kata Containers payload for Confidential Containers
+on:
+  push:
+    branches:
+      - CCv0
+jobs:
+  build-asset:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        asset:
+          - cc-cloud-hypervisor
+          - cc-kernel
+          - cc-qemu
+          - cc-rootfs-image
+          - cc-shim-v2
+          - cc-virtiofsd
+          - cc-sev-kernel
+          - cc-sev-ovmf
+          - cc-sev-rootfs-initrd
+          - cc-tdx-kernel
+          - cc-tdx-qemu
+          - cc-tdx-td-shim
+          - cc-tdx-tdvf
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install docker
+        run: |
+          curl -fsSL https://test.docker.com -o test-docker.sh
+          sh test-docker.sh
+
+      - name: Build ${{ matrix.asset }}
+        run: |
+          ./tools/packaging/kata-deploy/local-build/kata-deploy-copy-yq-installer.sh
+          ./tools/packaging/kata-deploy/local-build/kata-deploy-binaries-in-docker.sh --build="${KATA_ASSET}"
+          build_dir=$(readlink -f build)
+          # store-artifact does not work with symlink
+          sudo cp -r "${build_dir}" "kata-build"
+        env:
+          KATA_ASSET: ${{ matrix.asset }}
+          TAR_OUTPUT: ${{ matrix.asset }}.tar.gz
+
+      - name: store-artifact ${{ matrix.asset }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: kata-artifacts
+          path: kata-build/kata-static-${{ matrix.asset }}.tar.xz
+          if-no-files-found: error
+
+  create-kata-tarball:
+    runs-on: ubuntu-latest
+    needs: build-asset
+    steps:
+      - uses: actions/checkout@v3
+      - name: get-artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: kata-artifacts
+          path: kata-artifacts
+      - name: merge-artifacts
+        run: |
+          ./tools/packaging/kata-deploy/local-build/kata-deploy-merge-builds.sh kata-artifacts
+      - name: store-artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: kata-static-tarball
+          path: kata-static.tar.xz
+
+  kata-payload:
+    needs: create-kata-tarball
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: get-kata-tarball
+        uses: actions/download-artifact@v3
+        with:
+          name: kata-static-tarball
+      - name: build-and-push-kata-payload
+        id: build-and-push-kata-payload
+        run: |
+          docker login -u ${{ secrets.COCO_QUAY_DEPLOYER_USERNAME }} p ${{ secrets.COCO_QUAY_DEPLOYER_PASSWORD }} quay.io
+          ./tools/packaging/kata-deploy/local-build/kata-deploy-build-and-upload-payload.sh $(pwd)/kata-static.tar.xz "quay.io/repository/confidential-containers/runtime-payload-ci" "kata-containers-latest"
```

Comment:
This is probably not helpful, but I'm not sure about `kata-containers-latest` as the tag. I with there is merit to having a latest 'release' and a latest 'CI' build, so if we do adopt that I'd want to let users be clear which is which. I'm not sure if this makes sense and is fair?

Reply:
That's a totally fair question.

So, a few things to keep in mind here.
1. Although we're using `quay.io/confidential-containers` as the place to store the payload images, we do use different registries / repos / whatever it's called for **official** and **CI** images.
  * **official**: `quay.io/confidential-containers/runtime-payload`
  * **CI**: `quay.io/confidential-containers/runtime-payload-ci`
2. I see this also being used by `enclave-cc`, thus just tagging `latest` could cause confusion.  Thus I've gone for `kata-containers-latest` and I hope `enclave-cc` would go for  `enclave-cc-latest` on their side.
3. I do **not** expect users to be using anything from the `quay.io/confidential-containers/runtime-payload-ci` registry, that's meannt for developers, for testing and CI purpose.

With everything said above, do you still think we'll have a confusion even with the "repos" / registries being different?

Reply:
Cool - thanks for that explanation. I did spot that is was in a slightly different repo (with `-ci` appended), but then promptly forgot after looking at the rest of the PR. I think this is totally fine to start with and we can always adjust later if the use case come up rather than trying to predict what might happen. Sorry for the noise!

Reply:
Just added a description for the repository, see https://quay.io/repository/confidential-containers/runtime-payload-ci?tab=info .